script.js

```diff
@@ -0,0 +1,493 @@
+    keyBoard.innerHTML += `<div class="key key-${key}">${keysRuChar[key]}</div>`;
+    document.querySelector('div.wrapper').append(keyBoard);
+  }
+}
+
+function textFocus() {
+  const textarea = document.querySelector('textarea');
+  textarea.focus();
+  textarea.addEventListener('blur', (e) => {
+    e.target.focus();
+  });
+}
+
+function createText() {
+  const text = document.createElement('div');
+  text.className = 'text';
+  text.innerHTML = 'Клавиатура создавалась в Windows <br> Комбинация клавиш для переключения языка: ';
+  document.querySelector('div.wrapper').append(text);
+}
+
+function activeButton() {
+  const btn = document.querySelectorAll('div.key');
+  const textarea = document.querySelector('textarea');
+  const textareaSelectionStart = textarea.selectionStart;
+  const textareaSelectionEnd = textarea.selectionEnd;
+  btn.forEach((el) => {
+    el.addEventListener('mousedown', (e) => {
+      btn.forEach((el) => {
+        el.classList.remove('activeClick');
+      });
+      el.classList.add('activeClick');
+      if (el.innerHTML === 'Enter') {
+        textarea.value += '\n';
+      } else if (el.innerHTML === 'Tab') {
+        textarea.value += '\t';
+      } else if (el.innerHTML === 'Space') {
+        textarea.value += ' ';
+      } else if (el.innerHTML === 'Ctrl') {
+      } else if (el.innerHTML === 'CapsLock') {
+      } else if (el.innerHTML === 'Alt') {
+      } else if (el.innerHTML === 'Shift') {
+        console.log('shift');
+      } else if (el.innerHTML === 'Win') {
+      } else if (el.innerHTML === 'Backspace') {
+        textarea.value = textarea.value.slice(0, -1);
+      } else if (el.innerHTML === 'Del') {
+      } else {
+        textarea.value += el.innerHTML;
+      }
+    });
+  });
+  btn.forEach((el) => {
+    el.addEventListener('mouseup', () => {
+      btn.forEach((el) => {
+        el.classList.remove('activeClick');
+      });
+    });
+  });
+}
+
+function pressButton() {
+  const input = document.querySelector('div.text-input');
+  const textarea = document.querySelector('textarea');
+  input.addEventListener('keydown', (el) => {
+    el.preventDefault();
+
+    if (el.shiftKey === true) {
+      pressUppercase();
+    }
+    
+    let count = 0;
+    for (key in keysEn) {
+      if (key === event.code) {
+        const btn = document.querySelectorAll('div.key');
+        btn[count].classList.add('active');
+        btn[count].classList.add('activeClick');
+        if (keysEn[key] === 'Enter') {
+          textarea.value += '\n';
+        } else if (keysEn[key] === 'Shift') {
+        } else if (keysEn[key] === 'Tab') {
+          textarea.value += '\t';
+        } else if (keysEn[key] === 'Space') {
+          textarea.value += ' ';
+        } else if (keysEn[key] === 'Ctrl') {
+        } else if (keysEn[key] === 'Alt') {
+        } else if (keysEn[key] === 'Del') {
+        } else if (keysEn[key] === 'CapsLock') {
+        } else if (keysEn[key] === 'Backspace') {
```

Comment:
better to create an array and use the method "some", misuse of conditional operator